Scotland_Yard_Game.gs

```diff
@@ -1051,6 +1051,8 @@ function gameplay(){
   
 }
 
+//Thank you for reviewing my code
```

Comment:
![](https://media.giphy.com/media/3o85xwxr06YNoFdSbm/giphy.gif)

Reply:
@azharkhan , thanks for taking time bro. Appreciate it.